server/webhook_worker.go

```diff
@@ -49,7 +51,40 @@ func (ww webhookWorker) process(msg *webhookMessage) (err error) {
 		return err
 	}
 
-	channelsSubscribed, err := ww.p.getChannelsSubscribed(v, msg.InstanceID)
+	// To check if this is comment-related webhook payload
+	isCommentEvent := wh.Events().Intersection(commentEvents).Len() > 0
+	visibilityAttribute := ""
+	if isCommentEvent {
+		// We will only send webhook events if we have a connected instance.
+		instance, err := ww.p.instanceStore.LoadInstance(msg.InstanceID)
```

Comment:
whats the need of loading instance if you are only usings its id afterwards and we already have the instanceID from the msg

Reply:
to get the client